packages/expo-modules-core/CHANGELOG.md

```diff
@@ -4,12 +4,17 @@
 
 ### 🛠 Breaking changes
 
+- Remove deprecated `RCTDeviceEventEmitter`. ([#24298](https://github.com/expo/expo/pull/24298) by [@EvanBacon](https://github.com/EvanBacon))
+- Remove `deprecate` utility function. ([#24298](https://github.com/expo/expo/pull/24298) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
To be honest I don't think these are breaking changes for other SDK packages and also were not documented anywhere so people shouldn't have used them.
App's code shouldn't even import from `expo-modules-core` as it should only be a transitive dependency of the `expo` package.